src/api/app/controllers/person/token_controller.rb

```diff
@@ -7,25 +7,25 @@ class TokenController < ApplicationController
     def index
       authorize @user, :show?
 
-      @list = @user.service_tokens
+      @list = @user.tokens
     end
 
     # POST /person/<login>/token
     def create
       authorize @user, :update?
 
-      pkg = nil
-      if params[:project] || params[:package]
-        pkg = Package.get_by_project_and_name(params[:project], params[:package])
-      end
-      @token = Token::Service.create(user: @user, package: pkg)
+      pkg = if params[:project] || params[:package]
```

Comment:
[nitpick] this refactoring is unrelated

Reply:
yes, since we were refactoring other things, this nitpick isn't really relevant.